src/libs/actions/Task.ts

```diff
@@ -653,7 +653,8 @@ function setAssigneeValue(assigneeEmail: string, assigneeAccountID: number, shar
     let chatReport: OnyxEntry<OnyxTypes.Report> = null;
 
     if (!isCurrentUser) {
-        chatReport = ReportUtils.getChatByParticipants([assigneeAccountID]);
+        const reportID = shareDestination;
```

Comment:
The param name `shareDestination` is a reportID? If so, can we rename this to something like `shareToReportID`?